src/components/editions/avatar/index.tsx

```diff
@@ -29,6 +29,10 @@ const Avatar: FC<Props> = ({ item }) => {
 	const [contributor] = item.contributors;
 	const format = getFormat(item);
 
+	if (!contributor) {
+		return null;
+	}
```

Comment:
Thanks ESLint... that's not actually true. Although tbf it's actually TS that's being economical with the truth here.

@joecowton1 you might find this useful here:

```ts
const headOption = <A>(arr: A[]): Option<A> =>
  arr.length > 0 ? some(arr[0]) : none;
```

Reply:
Ah thanks Jamie! Was just scratching my head for a moment there.